src/main/java/com/artipie/asto/rx/RxCopy.java

```diff
@@ -67,6 +66,15 @@ public RxCopy(final RxStorage from, final List<Key> keys) {
      * @return The completion signal.
      */
     public Completable copy(final RxStorage to) {
-        return Completable.error(new IllegalStateException("not implemented"));
+        return Completable.merge(
+            this.keys.stream()
```

Comment:
@Sammers21 I don't like this mix of streams API and RxJava. I'd change it to:
```java
Observable.fromIterable(keys).flatMapCompletable(
  key -> from.value(key).flatMapComplatable(content -> to.save(key, content))
);
```

Reply:
What is wrong with the "mix"?